tests/java-db-test/src/main/webapp/script.js

```diff
@@ -0,0 +1,4 @@
+// This function will show ids and names of all users in the database in the console.
```

Comment:
I would update this comment to be more generalized, in case someone wants to test something other than the users query

```suggestion
// This function runs the query in the Test Servlet and logs the results in the console.
```

Reply:
done